<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QWidget" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>1181</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="main_layout">
    <item>
     <widget class="QGraphicsView" name="deck_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="screen_layout">
      <item>
       <widget class="DeckBuilder" name="view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="button_layout">
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_button">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_button">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="gui_layout">
      <item>
       <layout class="QFormLayout" name="form_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="name_label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="name_input"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="type_label">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="type_input"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="civ_label">
          <property name="text">
           <string>Civilization</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="civ_input"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="cost_label">
          <property name="text">
           <string>Cost</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="cost_input"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="power_label">
          <property name="text">
           <string>Power</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="power_layout">
          <item>
           <widget class="QComboBox" name="power_option"/>
          </item>
          <item>
           <widget class="QLineEdit" name="power_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="search_button">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeckBuilder</class>
   <extends>QGraphicsView</extends>
   <header>deckbuilder.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
